include/nuttx/timers/timer.h

```diff
@@ -116,6 +116,7 @@ struct timer_notify_s
 {
   struct sigevent event;    /* Describe the way a task is to be notified */
   pid_t           pid;      /* The ID of the task/thread to receive the signal */
+  bool            oneshot;  /* Single notification or periodic notifications */
```

Comment:
@xiaoxiang781216 I made `oneshot` extension instead of `periodic` not to introduce a "breaking" change, but it appears that apps examples are broken because `struct timer_notify_s` is allocated on stack and memory is not cleared before set.
Taking into account this I will issue a PR to change `openshot` to `periodic`. What do you think?

Reply:
It's fine too me.